test/test_ecco.jl

```diff
@@ -38,14 +38,67 @@ using Dates
         @test Ny == size(temperature)[2]
         @test Nz == size(temperature)[3]
         @test Nt == size(temperature)[4]
+
+        @test ECCO_fts.times[1]   == ECCO_times(temperature[1])
+        @test ECCO_fts.times[end] == ECCO_times(temperature[end])
     end
 end
 
+@testset "LatitudinallyTaperedPolarMask" begin
+
+    grid = LatitudeLongitudeGrid(size = (100, 100, 10), latitude = (-75, 75), longitude = (0, 360), z = (-200, 0))
+    
+    φ₁ = grid.φᵃᶜᵃ[1]
+    φ₂ = grid.φᵃᶜᵃ[21]
+    φ₃ = grid.φᵃᶜᵃ[80]
+    φ₄ = grid.φᵃᶜᵃ[100]
+    z₁ = grid.zᵃᵃᶜ[6]
+
+    mask = LatitudinallyTaperedPolarMask(northern_edges = (φ₃, φ₄), 
+                                         southern_edges = (φ₁, φ₂), 
+                                         z_edges = (z₁, 0))
+
+    t_restoring = ECCORestoring(:temperature, CPU(); 
+                                dates, 
+                                mask, 
+                                rate = 1 / 1000.0,
+                                maxiter = 1)
```

Comment:
should the max iter be set to the order of the advection scheme? or something like that?

Reply:
I am not sure. I don't think it has to do with the model detail, it has more to do with the discrepancy of the bathymetry we use with the ECCO bathymetry